README.md

```diff
@@ -56,6 +44,24 @@ If you want to contact us, e.g. before starting a more complex feature,
 you can join us at
 [#owncloud-client-dev](irc://irc.freenode.net/#owncloud-client-dev).
 
+## Maintainers and Contributors
+
+The current maintainers of this repository are:
+
+* Markus Goetz <guruz@owncloud.com>
+* Olivier Goffart <ogoffart@owncloud.com>
+* Christian Kamm <mail@ckamm.de>
+
+ownCloud Desktop Client is developed by the ownCloud community and receives
+patches from a variety of authors.
+
+Past maintainers:
+
+* Klaas Freitag <freitag@owncloud.com>
+* Daniel Molkentin <danimo@owncloud.com>
+* Jocelyn Turcotte <jturcotte@woboq.com>
```

Comment:
That address isn't valid anymore. You can just remove me from the list, the git log is paying tribute enough for my need.